Model/JWTRequest.php

```diff
@@ -72,7 +72,12 @@ private function buildAuthHeader($method, $restUrl)
 
     private function buildURL($restUrl)
     {
-        return $this->tenant->getBaseUrl().$restUrl;
+        // Jira return absolute self links, so its more easy to work with get with absolute urls in such cases  
```

Comment:
Yeah, I get this :)

I mean, what is the "business case"? If you're passing absolute URL as argument to the method which will simply return it, why not to use this `$restUrl` in the first place without calling the method?

For instance, somewhere in a controller, instead of calling `buildUrl($absoluteUrl)` you can simply use this `$absoluteUrl`, right?